modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContextImpl.java

```diff
@@ -1094,6 +1095,10 @@ private Object _getValue(
 			ObjectField objectField = _objectFieldLocalService.getObjectField(
 				GetterUtil.getLong(ddmFormField.getProperty("objectFieldId")));
 
+			if (objectField.isLocalized()) {
+				return values.get(objectField.getI18nObjectFieldName());
+			}
+
```

Comment:
The object field types that can be localized are: Text, Long Text and Rich Text. I don't think that create a method for each one of this types is a good approach because the code of the method will be the same for all.

I though about move this to the [default method](https://github.com/liferay/liferay-portal/blob/36437f1b0cd38f2f367730e2854e74a2ea496cb6/modules/apps/object/object-api/src/main/java/com/liferay/object/field/business/type/ObjectFieldBusinessType.java#L63-L68), but it doesn't seems good for me because not every field can be localized.

What do you recommend in this case?
@marcelabc

Reply:
We talked about adding this logic in the method getDisplayContextValue() in the interface ObjectFieldBusinessType.